corehq/apps/hqcase/export.py

```diff
@@ -56,7 +60,7 @@ def get_matching_owner(case):
             return get_owner_id(case)
 
     def might_be_relevant(case):
-        return not filter_group or get_owner_id(case) in owner_ids
+        return get_owner_id(case) in owner_ids
```

Comment:
does this change make it so that if you select unknown users you don't actually get all data? i think that owner_ids from unknown users aren't guaranteed to be in the lists of users and groups